step-templates/redis-uninstall.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Id": "959fbb4a-8e2e-4e21-bc55-863837b3cbed",
+  "Name": "Redis - Uninstall service",
+  "Description": "This step can be used before unpacking a package with Redis service to stop and remove the previous installation, if there is one.",
+  "ActionType": "Octopus.Script",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Properties": {
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptBody": "$name = $OctopusParameters['Redis Service name']\r\n\r\n$service = Get-Service -Name $name -ErrorAction SilentlyContinue\r\n\r\nif (-not $service) {\r\n    Write-Host \">>> $name service not found. Skipping this time.\"\r\n    return\r\n}\r\n\r\nWrite-Host \">>> Uninstalling with: sc.exe\"\r\nif ($service.Status -eq 'Running') {\r\n    &\"sc.exe\" stop $name | Write-Host\r\n}\r\n&\"sc.exe\" delete $name | Write-Host\r\n\r\n$limit = 15\r\nwhile (Get-Service -Name $name -ErrorAction SilentlyContinue) {\r\n    Start-Sleep -s 1\r\n    \r\n    $limit = $limit - 1\r\n    if ($limit -eq 0) {\r\n        Throw \"Unable to stop Redis service within 15s\"\r\n    }\r\n}",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "12713171-c487-4118-8a4c-7bd8c2e5e269",
+      "Name": "Redis Service name",
```

Comment:
Could you please remove the spaces in the parameter names and prefix with `rus_`

Reply:
Done. I was not sure if this is really required since existing templates rarely conform to this rule.

Reply:
Correct, this is a pretty new requirement from our side to limit the risk of duplicate parameters between step templates and variables in projects/library variables sets

Reply:
Looks like this one still needs to be updated with prefix and removal of spaces

Reply:
Apologies, I had hit commit before saving the file. Now its updated